lib/Requester/Decorator/Authentication/OAuthClientTokenAuthenticator.js

```diff
@@ -0,0 +1,180 @@
+import Headers from '../../Headers';
+import NoTokenAvailableException from '../../../Exception/NoTokenAvailableException';
+
+export default class OAuthClientTokenAuthenticator {
```

Comment:
Generic subnamespace `Fazland.Atlante.Decorator.Authentication.OAuth` instead having the prefix in the class name